<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>703</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Журнал посещений</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/new/prefix1/13.ico</normaloff>:/new/prefix1/13.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Отметить присутсвующих</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="currentTime">
        <property name="text">
         <string>09:30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentLesson">
        <property name="text">
         <string>Физика, лекция</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentAudit">
        <property name="text">
         <string>Университетский проспект, д. 26, лит. А, 2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Войти как преподователь</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Предмет</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lecture_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Преподаватель</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="teacher_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Группа</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="team_lineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Сортировать по </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <item>
             <property name="text">
              <string>Предмету</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Преподователю</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Группе</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search_lecture">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="lecture_tableWidget">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Предмет</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Преподаватель</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Группы</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>ФИО</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFio"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Группа</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditGroupStudent"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="infoLesson">
            <property name="text">
             <string>Предмет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoLecturer">
            <property name="text">
             <string>Преподаватель</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoGroup">
            <property name="text">
             <string>Группы</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Сортировать по </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>Студенту</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Группе</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchStudent">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string>ст1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ст2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ст3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ст4</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>02.09.2020</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>05.09.2020</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
